consensus/src/config.rs

```diff
@@ -66,6 +66,11 @@ pub fn is_emergency_iter(iter: u8) -> bool {
     iter >= EMERGENCY_MODE_ITERATION_THRESHOLD
 }
 
+/// Returns if the next iteration generator needs to be excluded
+pub fn exclude_next_generator(iter: u8) -> bool {
```

Comment:
`exlcude_next_generator` does not exclude anything?

Should be `is_last_iteration`?

Reply:
I don't want to know if this is the last iteration.
I want to know it we should exclude the next iteration generator from the current committees.

They can be the same, I agree, but the concept is different.
Luckily the comment state this explicitly